package.json

```diff
@@ -1,73 +1,48 @@
 {
   "name": "acrolinx-app-demo-select-ranges",
+  "private": true,
+  "version": "0.1.2",
+  "type": "module",
   "license": "Apache-2.0",
   "author": "Acrolinx",
   "url": "https://www.acrolinx.com",
-  "version": "0.1.2",
-  "private": true,
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
+    "test": "vitest run",
+    "format:check": "prettier . --check",
+    "format": "prettier . --write"
+  },
   "dependencies": {
-    "@acrolinx/app-sdk": "^0.1.1",
-    "@types/lodash": "^4.14.149",
-    "@types/react": "^16.9.17",
-    "@types/react-dom": "^16.9.4",
+    "@acrolinx/app-sdk": "^0.1.3",
     "lodash": "^4.17.21",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0"
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1"
   },
   "devDependencies": {
-    "@types/chromedriver": "^2.38.0",
-    "@types/fs-extra": "^8.0.1",
-    "@types/jest": "^24.0.25",
-    "@types/selenium-webdriver": "^4.0.6",
-    "browser-sync": "^2.26.7",
-    "chromedriver": "^83.0.0",
-    "dotenv": "^8.2.0",
-    "fs-extra": "^8.1.0",
-    "jest": "^24.9.0",
-    "jest-junit": "^10.0.0",
-    "react-scripts": "^3.4.0",
-    "selenium-webdriver": "^3.6.0",
-    "ts-jest": "^24.3.0",
-    "typescript": "3.7.4"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "startHttps": "HTTPS=true react-scripts start",
-    "build": "react-scripts build",
-    "test": "jest",
-    "eject": "react-scripts eject",
-    "tsc": "tsc"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
-  "homepage": "./",
-  "jest": {
-    "setupFilesAfterEnv": [
-      "./test/test-utils/jest-screen-shooter/jest.setup.ts"
-    ],
-    "transform": {
-      "^.+\\.tsx?$": "ts-jest"
-    },
-    "reporters": [
-      "default",
-      "jest-junit"
-    ]
-  },
-  "jest-junit": {
-    "outputDirectory": "tmp",
-    "outputName": "junit.xml"
+    "@types/chromedriver": "^81.0.5",
+    "@types/lodash": "^4.17.6",
+    "@types/node": "^20.14.10",
+    "@types/react": "^18.3.3",
+    "@types/react-dom": "^18.3.0",
+    "@types/selenium-webdriver": "^4.1.24",
+    "@typescript-eslint/eslint-plugin": "^7.13.1",
+    "@typescript-eslint/parser": "^7.13.1",
+    "@vitejs/plugin-react": "^4.3.1",
+    "browser-sync": "^3.0.2",
+    "chromedriver": "^126.0.4",
+    "dotenv": "^16.4.5",
+    "eslint": "^8.57.0",
```

Comment:
latest version is 9.6.0, could we use that?

Reply:
Lets do the eslint as another PR